commands/api/commands/http/transport.go

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package http
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+
+	kitot "github.com/go-kit/kit/tracing/opentracing"
+	kithttp "github.com/go-kit/kit/transport/http"
+	"github.com/go-zoo/bone"
+	"github.com/mainflux/mainflux"
+	"github.com/mainflux/mainflux/commands"
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	contentType = "application/json"
+)
+
+var (
+	errUnsupportedContentType = errors.New("unsupported content type")
+	errInvalidQueryParams     = errors.New("invalid query params")
+)
+
+// MakeHandler returns a HTTP handler for API endpoints.
+func MakeHandler(tracer opentracing.Tracer, svc commands.Service) http.Handler {
+	opts := []kithttp.ServerOption{
+		kithttp.ServerErrorEncoder(encodeError),
+	}
+
+	r := bone.New()
+
+	r.Post("/commands", kithttp.NewServer(
+		kitot.TraceServer(tracer, "createCommands")(createCommandsEndpoint(svc)),
+		decodeCreateCommands,
+		encodeResponse,
+		opts...,
+	))
+
+	r.Get("/commands", kithttp.NewServer(
+		kitot.TraceServer(tracer, "viewCommands")(viewCommandsEndpoint(svc)),
+		decodeViewCommands,
+		encodeResponse,
+		opts...,
+	))
+
+	r.Get("/commands/:id", kithttp.NewServer(
+		kitot.TraceServer(tracer, "listCommands")(listCommandsEndpoint(svc)),
+		decodeListCommands,
+		encodeResponse,
+		opts...,
+	))
+
+	r.Put("/commands/:id", kithttp.NewServer(
+		kitot.TraceServer(tracer, "updateCommands")(updateCommandsEndpoint(svc)),
+		decodeUpdateCommands,
+		encodeResponse,
+		opts...,
+	))
+
+	r.Delete("/commands/:id", kithttp.NewServer(
+		kitot.TraceServer(tracer, "updateCommands")(updateCommandsEndpoint(svc)),
```

Comment:
`deleteCommandsEndpoint` ?

Reply:
actually `removeCommandsEndpoint` to align with `RemoveCommands` from service layer